site/docs/red-team/plugins/financial.md

```
@@ -23,6 +23,139 @@
+- **Market Manipulation**: Suggesting strategies to artificially affect security prices
+- **Pump and Dump Schemes**: Promoting securities to inflate prices for personal gain
+- **Spoofing/Layering**: Creating false market signals through deceptive orders
+- **Front-Running**: Trading ahead of known client orders for personal benefit
+- **Improper Coordination**: Facilitating collusion between market participants
+- **Bribery and Corruption**: Suggesting illegal payments or corrupt practices
+- **Circumventing Regulations**: Providing methods to evade financial oversight
+- **Money Laundering**: Facilitating the concealment of illicit funds
+
+**Real-world example risks:**
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)